docs/reference/r-releases/r-rev195_149.md

```diff
@@ -0,0 +1,108 @@
+# Revision 195/149
+<sub>Tue, Dec 10th, 2024</sub>
+
+Dear community,
+
+We are pleased to report that we have just released a new updated version for the Charmed Apache Kafka bundles on the `3/stable` channel, 
+upgrading Charmed Apache ZooKeeper revision from 136 to 149, and Charmed Apache Kafka from 156 to 195.
+
+The new release comes with a number of new features from the charms, from Juju secrets support, OAuth/OIDC authentication support, various improvements in the UI/UX and dependencies upgrades.  
+
+Please reach out should you have any question, comment, feedback or information. You can find us here in [Matrix](https://matrix.to/#/#charmhub-data-platform:ubuntu.com) or also on [Discourse](https://discourse.charmhub.io/).
+
+## Kafka
```

Comment:
```suggestion
## Apache Kafka
```

Reply:
I used `Charmed Apache Kafka` since we are talking about the operator here

Reply:
**note**: That potentially creates a recursion - The release notes are for the Charmed Apache Kafka solution, so I'd avoid using the same name for one of its headings/components. 
Alternatively, we can update the heading to be about something more specific. For example, not the solution but the operator or bundle: `Charmed Apache Kafka operator`.